src/nifgen/metadata/attributes_addon.py

```diff
@@ -50,6 +50,24 @@
     1050103: { "codegen_method": "no" },  # ERROR_ELABORATION - IVI Attribute - #824
     1050003: { "codegen_method": "no" },  # QUERY_INSTR_STATUS - IVI Attribute - #824
     1050004: { "codegen_method": "no" },  # CACHE - IVI Attribute - #824
+    1150244: { "codegen_method": "no" },  # DIRECT_DMA_ENABLED - nifgen review - StreamStor not supported - #858
```

Comment:
I think "nifgen review" is more of a historical snippet and source control encodes that well.
You can remove that.